components/prism-io.js

```diff
@@ -0,0 +1,31 @@
+Prism.languages.io = {
+    'comment': [
+        {
+            pattern: /(^|[^\\])\/\*[\s\S]*?(?:\*\/|$)/,
+            lookbehind: true
+        },
+        {
+            pattern: /(^|[^\\:])\/\/.*/,
+            lookbehind: true
+        },
+        {
+            pattern: /(^|[^\\:])#.*/,
+            lookbehind: true
+        }
+    ],
+    'triple-quoted-string': {
+        pattern: /"""(?:\\.|(?!""")[^\\])*"""/,
+        greedy: true,
+        alias: 'string'
+    },
+    'string': {
+        pattern: /"(?:\\.|(?!")[^\\\r\n])*"/,
```

Comment:
You can remove the negative look-ahead if you include `"` in the negated character class.